app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionDetailViewModel.java

```diff
@@ -88,12 +88,12 @@ private void setSession(@NonNull Session session) {
         this.languageResId = session.lang != null ? decideLanguageResId(session.lang.toUpperCase()) : R.string.lang_en;
     }
 
-    public Maybe<Session> findSession(int sessionId) {
+    public Completable findSession(int sessionId) {
         final String languageId = Locale.getDefault().getLanguage().toLowerCase();
-        return sessionsRepository.find(sessionId, languageId)
-                .map(session -> {
+        return  sessionsRepository.find(sessionId, languageId)
+                .flatMapCompletable(session -> {
                     setSession(session);
```

Comment:
Would you like the simple name ``Completable loadSession(sessionId)`` ?
``load`` means read session and sets member.
This contains multiple stuffs but I think enough in this case because only called from ``SessionDetailFragment.onCreate()``.